src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -36,7 +41,9 @@ class ReportController(actionContext: ActionContext,
         {
             "/reports/$reportName/versions/$version/artefacts/$focalArtefact?inline=true"
         }
-        return ReportViewModel(reportDetails.copy(displayName = displayName), focalArtefactUrl, context)
+
+        val isAdmin = context.hasPermission(ReifiedPermission("reports.review", Scope.Global()))
```

Comment:
Need a test for this

Reply:
https://github.com/vimc/orderly-web/pull/45/files#diff-0a3fddc91043e8c897a7e5dc942093b8